recipes/tzlocal/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.2.2" %}
+
+package:
+  name: tzlocal
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/regebro/tzlocal.git
+  git_rev: {{ version }}
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
Please add the arguments ` --single-version-externally-managed --record=record.txt`.